lib/eval-script.js

```diff
@@ -13,12 +15,12 @@ module.exports = function(el) {
 
   script.type = "text/javascript"
 
-  if (src != "") {
-    script.src = src;
-    script.async = false; // force synchronous loading of peripheral js
+  if (src !== "") {
+    script.src = src
+    script.async = false // force asynchronous loading of peripheral js
```

Comment:
@robinnorth I'm confused. Why did you change this back? I thought we agreed it was synchronous.

Reply:
Oops, I guess I didn't quite successfully resolve that merge conflict... Will fix.